requirements.txt

```diff
@@ -7,3 +7,5 @@ polling
 -e git+https://github.com/ebi-ait/ingest-client.git#egg=hca_ingest
```

Comment:
I am not sure I understand why we are based on the ingest-client repo rather than on a concrete version.

Reply:
Because the work to increment the version number or even set the correct repo location has not been done in ingest client for a long time. We should priorities this soon, or we can make it contingent on this PR.

Reply:
Because the CI CD it seems I'm going to have to fix ingest-client first anyway.
https://gitlab.ebi.ac.uk/hca/ingest-archiver/-/jobs/436739#L346

Reply:
I've manually locked the version of ingest client, which has worked thankfully.
I've opened [a ticket for the issues using pip-compile with ingest-client](https://github.com/ebi-ait/dcp-ingest-central/issues/306), but they are no longer blocking this merge.